### ISSUE TITLE
Transaction broadcast problem

### ISSUE THREAD

**Issue by mahsumurebe** [wavesplatform/Waves#2893]
---
name: Bug report
about: Broadcast problem
title: 'Transaction broadcast problem'
labels: bug
assignees:

---

Waves v1.1.6 wallet application is installed on my server. There is a sufficient amount of coin at the address I am trying to transaction. But when I try to broadcast a transaction, "Error is unknown" error occurs.

1. HTTP POST /transactions/sign

    POST BODY:
    ```json
    {
      "type": 4,
      "sender": "3PQmdQUAQqGkiFg3HDhmqc5VjgBrPucK1FK",
      "recipient": "3PDpLfVW7FhGU11MkCAAEvKzkcWwvpekArZ",
      "amount": 100000000,
      "attachment": "",
      "senderPublicKey": null,
      "assetId": "2ad6RZP9DVntNGNkS5so4oax2x2yZV4ZTr2yCNrpj1VN",
      "fee": 100000
    }
   ```

   RESPONSE HTTP 200:

   ```json
   {
     "senderPublicKey": "7QAVDbVCsVJR1nBWcqZcjbRVJj6TUck8Ba2uuXPQUy8Y",
     "amount": 100000000,
     "signature": "43cm4Zd8fQYfCtWmJsrCdjnkkcJzjzabZ72NUQpr2UpbyuBJNjwJQhR9trwvTNa3312uDpa5FthDJpDrHdairWhK",
     "fee": 100000,
     "type": 4,
     "version": 1,
     "attachment": "",
     "sender": "3PQmdQUAQqGkiFg3HDhmqc5VjgBrPucK1FK",
     "feeAssetId": null,
     "proofs": [
       "43cm4Zd8fQYfCtWmJsrCdjnkkcJzjzabZ72NUQpr2UpbyuBJNjwJQhR9trwvTNa3312uDpa5FthDJpDrHdairWhK"
     ],
     "assetId": "2ad6RZP9DVntNGNkS5so4oax2x2yZV4ZTr2yCNrpj1VN",
     "recipient": "3PDpLfVW7FhGU11MkCAAEvKzkcWwvpekArZ",
     "feeAsset": null,
     "id": "2we1NmXVgJedLmPzNMALHF4UK8QmDHzeqLvKd5MpRY7r",
     "timestamp": 1579797218875
   }
    ```
2. HTTP POST /transactions/broadcast

    POST BODY

    ```json
    {
      "senderPublicKey": "7QAVDbVCsVJR1nBWcqZcjbRVJj6TUck8Ba2uuXPQUy8Y",
      "amount": 100000000,
      "signature": "43cm4Zd8fQYfCtWmJsrCdjnkkcJzjzabZ72NUQpr2UpbyuBJNjwJQhR9trwvTNa3312uDpa5FthDJpDrHdairWhK",
      "fee": 100000,
      "type": 4,
      "version": 1,
      "attachment": "",
      "sender": "3PQmdQUAQqGkiFg3HDhmqc5VjgBrPucK1FK",
      "feeAssetId": null,
      "proofs": [
        "43cm4Zd8fQYfCtWmJsrCdjnkkcJzjzabZ72NUQpr2UpbyuBJNjwJQhR9trwvTNa3312uDpa5FthDJpDrHdairWhK"
      ],
      "assetId": "2ad6RZP9DVntNGNkS5so4oax2x2yZV4ZTr2yCNrpj1VN",
      "recipient": "3PDpLfVW7FhGU11MkCAAEvKzkcWwvpekArZ",
      "feeAsset": null,
      "id": "2we1NmXVgJedLmPzNMALHF4UK8QmDHzeqLvKd5MpRY7r",
      "timestamp": 1579797218875
    }
   ```

   RESPONSE HTTP 500

   ```json
   {
     "error": 0,
     "message": "Error is unknown"
   }
    ```

The transaction is expected to be broadcasted.
**Server:**
 - OS: Ubuntu
 - Version 18.04.3 bionic

This error occurs when I try to send any Asset or WAVES.

**Comment by mahsumurebe** [comment]
Any news ?

**Comment by msmolyakov** [comment]
Hi! This bug has already been fixed in the upcoming version 1.1.8.
Try to restart node or, as workaround right now, you can [build node from the latest sources](https://github.com/wavesplatform/Waves#mainnet) from the `version-0.17.x` branch.
Thanks for the report.

**Comment by mahsumurebe** [comment]
Thank you. I upgraded the node to version 1.1.7. It works now.
